src/views/CharacterListView.js

```diff
@@ -2,20 +2,24 @@ import React from "react";
 import { connect } from "react-redux";
 
 import { CharacterList } from "../components";
-// import actions
+import { getData } from "../actions";
 
 class CharacterListView extends React.Component {
-  constructor() {
-    super();
-  }
+ 
 
   componentDidMount() {
     // call our action
+    console.log(this.props.getData())
   }
 
   render() {
     if (this.props.fetching) {
```

Comment:
Here you have put `this.props.fetching` but in your `mstp` function, you wrote the property as `fatching`. Because of this, your `Fetching Data` message will never show.